portafly/src/components/util/AlertsContext.tsx

```diff
@@ -14,7 +14,7 @@ interface IAlert {
 }
 
 type IAlertsContext = { addAlert: (alert: IAlert) => void }
-const AlertsContext = React.createContext<IAlertsContext>({ addAlert: () => {} })
+const AlertsContext = React.createContext<IAlertsContext>({ addAlert: () => { } })
```

Comment:
Same here, and the rest of the PR. Even if it was auto-triggered, it should not pollute the reason of this PR

Reply:
This is not be triggered by our eslint ruleset anyway.

Reply:
done